packages/inspection-capture-web/src/PhotoCapture/hooks/usePhotoCaptureTutorial.ts

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { PhotoCaptureTutorialOption } from '@monkvision/types';
+import { useObjectMemo } from '@monkvision/common';
+
+export const STORAGE_KEY_PHOTO_CAPTURE_TUTORIAL = '@monk_photoCaptureTutorial';
+
+/**
+ * Enum of the different views of the PhotoCapture Tutorial component.
+ */
+export enum TutorialViews {
```

Comment:
I think we should rename this to `TutorialSteps`, I think it's clearer (you'll have to also switch from "view" to "step" in variable names, doc etc...)